openapi.yaml

```diff
@@ -1497,6 +1506,9 @@ paths:
                             Domain-specific fields for domains such as EO, SAR and point clouds.
                           * **Custom Properties**:
                             It is generally allowed to add custom fields.
+
+                          This property REQUIRES to add `datacube` to the list
+                          of `stac_extensions`.
```

Comment:
Do I understand this correctly:
as `summaries` is a required field, `datacube` is consequently a required item under `stac_extensions` (a field that is not required)? 
If so: shouldn't `stac_extesions` also be required with at least a `datacube` item?

Reply:
Ouh, good catch. I added this to the wrong description. Will move to `cube:dimensions`.